### PR TITLE
Removing compiler warnings

### PR DESCRIPTION
**Pull request by zelklen**
Removed compiler warnings in BasicShips.cpp

### CODE REVIEW

File: DancingPlutonium/IRng.cpp [hunk 1]
```diff
@@ -8,4 +8,4 @@ int RandomIntRange(const int& A, const int& B)
 	dist.param(std::uniform_int_distribution<int>::param_type(A, B));
 
 	return dist(twister);
-}
+}
```
Comment (MarioMigliacio) [inline DancingPlutonium/IRng.cpp]: i told you so

Comment (zelklen) [inline DancingPlutonium/IRng.cpp]: Undo all changes to this file. If only we knew how.

Comment (MarioMigliacio) [inline DancingPlutonium/IRng.cpp]: its ok man. its ok!

**Review by MarioMigliacio — APPROVED** [review]
It looks great! +1

**Comment by MarioMigliacio** [comment]
I am accepting the first pull request!

**Pull request merged**